DESCRIPTION

```diff
@@ -1,27 +1,29 @@
 Package: hubUtils
-Title: Utility functions for Infectious Disease Modeling Hubs
+Title: Utility Functions for Infectious Disease Modeling Hubs
 Version: 0.1.0
-Authors@R: 
-    c(person("Anna", "Krystalli", , "annakrystalli@googlemail.com", role = c("aut", "cre"),
+Authors@R: c(
+    person("Anna", "Krystalli", , "annakrystalli@googlemail.com", role = c("aut", "cre"),
            comment = c(ORCID = "0000-0002-2378-4915")),
-      person("Li", "Shandross", , "lshandross@umass.edu", role = "ctb"),
-      person("Nicholas G.", "Reich", , "nick@umass.edu", role = "ctb",
+    person("Li", "Shandross", , "lshandross@umass.edu", role = "ctb"),
+    person("Nicholas G.", "Reich", , "nick@umass.edu", role = "ctb",
            comment = c(ORCID = "0000-0003-3503-9899")),
-      person("Evan L.", "Ray", , "elray@umass.edu", role = "ctb"),
-      person(family = "Consortium of Infectious Disease Modeling Hubs",
-           role = c("cph")))
-Description: A set of utility functions for downloading, plotting, and scoring
-    forecast and truth data from Infectious Disease Modeling Hubs.
+    person("Evan L.", "Ray", , "elray@umass.edu", role = "ctb"),
+    person(, "Consortium of Infectious Disease Modeling Hubs", role = "cph")
+  )
+Description: A set of utility functions for downloading, plotting, and
+    scoring forecast and truth data from Infectious Disease Modeling Hubs.
 License: MIT + file LICENSE
-Encoding: UTF-8
-Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.3.1
 URL: https://github.com/hubverse-org/hubUtils,
     https://hubverse-org.github.io/hubUtils/
 BugReports: https://github.com/hubverse-org/hubUtils/issues
+Depends: 
+    R (>= 2.10)
 Imports:
     checkmate,
     cli,
+    curl,
+    fs,
+    gh,
```

Comment:
It seems like there are other packages that depend on them because if I try to remove them the checks start failing. (I think that package installation or building also fails), so I've just left them in

Reply:
Oh that is weird, and they are actually used in the code so it's not like it's other packages that depend on them 🤔

Reply:
I have a hunch as to why these notes are showing up:

The code _is_ used but because of the memoisation in `zzz.R`, R CMD check does not see them:

https://github.com/hubverse-org/hubUtils/blob/c8922c59d3371984242a15ac8a02fc492db80496/R/zzz.R#L1-L6

Reply:
https://github.com/hubverse-org/hubUtils/pull/156 should fix this. I've set it to merge into the `prepare-for-cran` branch.

Reply:
I've closed my PR in favor of #157.